src/selectors/entities/preferences.js

```diff
@@ -120,14 +120,18 @@ const getThemePreference = createSelector(
     }
 );
 
+const getDefaultThemeName = createSelector(getConfig, (config) => config.DefaultTheme || 'default');
```

Comment:
If you wanted, you could make this `getDefaultTheme` and just have it return `Preferences.THEMES[config.DefaultTheme || 'default']`